packages/expo-router/src/modal/Modal.tsx

```diff
@@ -64,6 +64,13 @@ export interface ModalProps extends ViewProps {
    * while **Android is limited to three**.
    */
   detents?: ModalConfig['detents'];
+  /**
+   * See {@link ScreenProps["unstable_sheetFooter"]}.
+   *
+   * Footer component that can be used alongside formSheet stack presentation style.
+   * Works only when `presentation` is set to `formSheet`.
+   */
+  footer?: ModalConfig['footer'];
```

Comment:
According to screens docs, this is Android only prop for now. Can you mark it here as well?

Also I think that it will be better to call in `unstable_footer`, following react-native-screens